### PR TITLE
main - required positional argument constantsModule

### PR DESCRIPTION
**Pull request by marcelotduarte**
issue #560 
introduced by commit 92a6f177a73f31a0bd4408096521a6d4d03b8949

### CODE REVIEW

File: cx_Freeze/main.py [hunk 1]
```diff
@@ -157,12 +157,13 @@ def ParseCommandLine():
...
     options = ParseCommandLine()
     executables = [cx_Freeze.Executable(options.script,
-                                        initScript = options.initScript,
-                                        base = options.baseName,
-                                        icon = options.icon,
-                                        targetName = options.targetName)
+                                        options.initScript,
+                                        options.baseName,
+                                        options.targetName,
+                                        options.icon)
```
Comment (anthony-tuininga) [inline cx_Freeze/main.py:163]: Why these changes? They don't have anything to do with the bug, do they? Can you remove them?

**Comment by anthony-tuininga** [comment]
I've just pushed a fix that resolves this issue in a cleaner way, I believe. So I'll close this. Thanks for the suggestion, however!

**Comment by varadhbhatnagar** [comment]
@anthony-tuininga Issue still comes when I install cx-Freeze from PyPI. I am using Ubuntu 18.04. Had to go back to 6.0.

**Comment by marcelotduarte** [comment]
@varadhbhatnagar
Go back to 6.0 or go ahead:
pip install -U git+https://github.com/anthony-tuininga/cx_Freeze.git@master

File: cx_Freeze/main.py (hunk 1 above)
Comment (MarkoShiva) [reply cx_Freeze/main.py:163]: Is there some special reason for not merging this commit? I have problem of fetching from pypi in the CI/CD environment. locally I patched the same file and it works. The problem is that on PyPi I don't see package updated and there is no development version available in pypi either so I can't rely on CI/CD to buld my executable for some projects. 
At least in my project nothing is broken by removing kwargs and making them args.

Comment (anthony-tuininga) [reply cx_Freeze/main.py:163]: Can you explain why switching from keyword args to positional args is an improvement? It may work just fine, but what is the advantage to making the change? If you can explain that and it makes sense to me I'll be happy to merge that commit!

Comment (MarkoShiva) [reply cx_Freeze/main.py:163]: Well I am certain that having a keyword arguments is nicer as you do not need to pay attention that much on names of parameters and their order. 
But however you had a grave bug in important project. You could either push update faster or accept marcelo commit.

Its sort of a big problem when Pypi start hosting projects as packages that fail. There should be checking for that automated for every working supported version of python. Imagine when I needed to edit a python package in venv to have it work with 3.8 and even then the CI/CD couldn't do that so I needed to downgrade.

I mean project is nice but its not good to leave people downloading your package from pypi that do not work for over a full month. I mean I still after more then a month of that issue have the same issue if I try to install normally 6.1 from PyPi. Will package there ever be updated?
